recipe_scrapers/yummly.py

```diff
@@ -11,12 +11,12 @@ def title(self):
         return self.soup.find("h1").get_text()
 
     def total_time(self):
-        return get_minutes(self.soup.find("div", {"class": "recipe-summary-item unit"}))
+        return int(
```

Comment:
Could you describe the reason for using an `int` cast rather than `get_yields`?

There is one other scraper - `IG` - that does [something similar](https://github.com/hhursev/recipe-scrapers/blob/faf354bd5735533cf9d5fb98d22785eed8c71b4a/recipe_scrapers/ig.py#L18) at the moment for the time field, but it seems to be rare across the codebase.

Reply:
@jayaddison because I was seeing this:

```
>>> from recipe_scrapers._utils import get_minutes
>>> get_minutes('23')
0
```